odoo/models.py

```diff
@@ -336,6 +336,11 @@ class BaseModel(MetaModel('DummyModel', (object,), {'_register': False})):
     as attribute.
     """
 
+    _depends = {}
+    """dependencies of models backed up by SQL views
+    ``{model_name: field_names}``, where ``field_names`` is any iterable
```

Comment:
Perhaps we can help developers understand what to expect with an extra sentence like:

_This is only used to determine the changes to flush to database before executing `search()` or `read_group()`. It won't be used for cache invalidation or recomputing fields_